docs/user-guide/src/version_support.md

```diff
@@ -3,79 +3,85 @@
 The Cluster API Provider Metal3 (CAPM3) team maintains the following branches for CAPM3 for different API versions.
 
 - CAPM3
-    - main
-        - v1beta1
-    - release-1.2
-        - v1beta1
-    - release-1.1
-        - v1beta1
+   - main
+      - v1beta1
+   - release-1.4
+      - v1beta1
+   - release-1.3
+      - v1beta1
+   - release-1.2
+      - v1beta1
+   - release-1.1
+      - v1beta1
 
 Currently, in Metal³ organization only CAPM3 and IPAM follow CAPI release cycles. The supported versions (excluding release candidates) for CAPM3 and IPAM releases are as follows:
 
 - CAPM3
-    - v1beta1
-        - v1.2.0, v1.1.3, v1.1.2, v1.1.1, v1.1.0
-    - v1alpha5
-        - v0.5.5, v0.5.4, v0.5.3, v0.5.2, v0.5.1, v0.5.0
+   - v1beta1
+      - v1.4.0, v1.3.2, v1.3.1, v1.3.0, v1.2.2, v1.2.1, v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0
 - IPAM
-    - v1alpha1
-        - v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
+   - v1beta1
+      - v1.4.0, v1.3.1, v1.3.0
+   - v1alpha1
+      - v1.2.2, v1.2.1, v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
```

Comment:
since you are removing v1alpha5 from CAPM3, you should also remove v0.1.2, v0.1.1, v0.1.0 from this line.

Reply:
done